packages/babel/package.json

```diff
@@ -44,7 +46,8 @@
   ],
   "peerDependencies": {
     "@babel/core": "^7.0.0",
-    "rollup": "^1.20.0||^2.0.0"
+    "rollup": "^1.20.0||^2.0.0",
+    "@types/babel__core": "^7.1.9"
```

Comment:
nit: those keys here should be sorted alphabetically

Reply:
👍  Done